dist/js/jquery.keyboard.js

```diff
@@ -3088,7 +3091,7 @@ http://www.opensource.org/licenses/mit-license.php
 	};
 
 	$keyboard.caret = function($el, param1, param2) {
-		if (!$el.length || $el.is(':hidden') || $el.css('visibility') === 'hidden') {
+		if (!$el || !$el.length || $el.is(':hidden') || $el.css('visibility') === 'hidden') {
```

Comment:
I didn't change this, must have been done automatically when running the grunt command, I only changed the js/jquery.keyboard.js file